airflow/dags/create_external_tables/airtable/external_airtable_california_transit_services.yml

```diff
@@ -288,3 +288,30 @@ schema_fields:
   - name: public_currently_operating_fixed_route
     type: STRING
     mode: NULLABLE
+  - name: holiday_website_status__from_provider_
```

Comment:
Perhaps we should leave this one out since we're using the `services.holiday_website_status` column now.